src/generator/backend/assets/css/schedule.css

```diff
@@ -0,0 +1,249 @@
+.session-list li {
+  cursor: pointer;
+}
+
+.session-list .label {
+  background: #ff8700;
+  border-color: #ebccd1;
+  color: #FFFFFF;
+  font-weight: 500;
+}
+
+.session-list .session-title {
+  margin-top: 0.1em;
+}
+
+.session-list .session-title .session-link {
+  font-size: 12px;
+}
+
+/* Adjustments for smaller screens */
+body {
+  font-size: 12px;
+}
+
+.session-list .label {
+  font-size: 80%;
+  margin-left: -8px;
+}
+
+.session-location {
+  text-align: left;
+}
+
+@media (min-width: 450px) {
+  body {
+    font-size: 14px
+  }
+
+  .session-list .label {
+    font-size: 90%;
+    padding: .2em .6em .3em;
+  }
+
+  .session-title {
+    float:left;
+    width: 75%;
+  }
+
+  .session-location {
+    float: right;
+    width: 25%;
+    text-align: right;
+  }
+}
+@media (min-width: 768px) {
+
+    #day-menu .dropdown-menu {
+        display: block;
+        opacity: 0;
+        top: -500px;
+        -moz-transition: opacity 500ms ease;
+        -webkit-transition: opacity 500ms ease;
+        -o-transition: opacity 500ms ease;
+        -ms-transition: opacity 500ms ease;
+        transition: opacity 500ms ease;
+    }
+
+    #day-menu > .dropdown-toggle:hover + .dropdown-menu {
+        display: block;
+        opacity: 1;
+        top: 50px;
+    }
+
+    #day-menu > .dropdown-toggle:hover {
+        background-color: #e7e7e7;
+    }
+
+    #day-menu .dropdown-menu:hover {
+        display: block;
+        opacity: 1;
+        top: 50px;
+    }
+}
+.btn-sign-up {
+  background: #ff8700;
+  color: #FFFFFF;
+  margin-left: 5px;
+  margin-top: -5px;
+}
+
+.video-link > .fa {
+	font-size: 22px;
+	color: #757575;
+}
+
+.video-link > .fa:hover {
+	color: #e52d27;
+}
+
+.video-link  {
+	margin-left: 3px;
+}
+
+.session-link > .fa {
+	font-size: 18px;
+	color: #757575;
+}
+
+.session-link > .fa:hover {
+	color: #2196F3;
+}
+
+.session-link + .inputbox {
+  font-size: 12px;
+}
+
+@media (max-width: 450px) {
+	.session-link > .fa {
+		font-size: 24px;
+	}
+	.video-link > .fa {
+		font-size: 28px;
+	}
+	.btn-sign-up {
+		font-size: 15px;
+		margin-top: -8px;
+	}
+	.session-title {
+		line-height: 1.5;
+	}
+}
+@media (max-width: 500px) {
+  .nav>li>a {
+    padding:8px !important;
+  }
+}
+.inputbox {
+  width:35%;;
+  padding: 4px;
+  margin-left: 5px;
+}
+::selection {
+  background-color: #ff8700;
+  color: #FFFFFF;
+}
+.footer {
+  bottom: 0;
+  margin: 0;
+  color: #777;
+  padding: 15px;
+  text-align: center;
+  }
+
+.large {
+  padding:3% ;
+  margin:1%;
+  display: block;
+  max-width:100%;
+  max-height:200px;
+  width: auto;
+  height: auto;
+}
+.medium {
+  padding:3% ;
+  margin:1%;
+  max-width:100% ;
+  max-height:140px;
+  width:auto;
+  height: auto;
+  display: block;
+}
+.small {
+  padding:3% ;
+  margin:1%;
+  max-width:100%;
+  max-height:120px;
+  width:auto;
+  height: auto;
+  display: block;
+}
+.sponsors {
+  position: relative;
+  display: block;
+  padding: 10px 15px;
+  margin: 25px 0px;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710180111/issues/source?bid=3233857&fileBranchId=3342834#l239)